city_scrapers/spiders/il_investment_policy.py

```diff
@@ -0,0 +1,157 @@
+
+        for item in response.xpath(
+            "//h2[text()='Agendas']/following-sibling::ul"
+            "[not(preceding-sibling::h2[text()='Minutes'])]"
+            "/li/p"
+        ):
+            meeting = Meeting(
+                title=self._parse_title(item),
+                description="",
+                classification=self._parse_classification(item),
+                start=self._parse_start(item),
+                end=None,
+                all_day=False,
+                time_notes=self._parse_time_notes(item),
+                location={
+                    "name": "James R. Thompson Center",
+                    "address": "100 W. Randolph St., Room 16-503, Chicago, Illinois"
+                },
+                source=response.url
+            )
+
+            meeting["status"] = self._get_status(meeting)
+            meeting["id"] = self._get_id(meeting)
+
+            day = date(meeting['start'].year, meeting['start'].month, meeting['start'].day)
+            meeting_type = self._parse_meeting_type(meeting['title'])
+            meeting['links'] = links.get((day, meeting_type), [])
+
+            yield meeting
+
+    @staticmethod
+    def _parse_title(item):
+        """Parse or generate meeting title."""
+        return item.xpath("text()[normalize-space(.)]").get().lstrip(' -').strip()
+
+    @staticmethod
+    def _parse_classification(
+        item, board_substrings=['board', 'iipb'], committee_substrings=['committee']
+    ):
+        """Parse or generate classification from allowed options."""
+        title = item.xpath("text()[normalize-space(.)]").get()
+        if any(substring in title.lower() for substring in board_substrings):
+            return BOARD
+        elif any(substring in title.lower() for substring in committee_substrings):
+            return COMMITTEE
+        return NOT_CLASSIFIED
+
+    @staticmethod
+    def _parse_meeting_type(
+        text,
+        ise_substrings=['ise', 'sudan', 'iran'],
+        ibr_substrings=['ibr', 'israel'],
+        iipb_substrings=['iipb', 'policy', 'board']
+    ):
+        if any(substring in text.lower() for substring in ise_substrings):
+            return IlInvestmentPolicyMeetingType.ISE
+        elif any(substring in text.lower() for substring in ibr_substrings):
+            return IlInvestmentPolicyMeetingType.IBR
+        elif any(substring in text.lower() for substring in iipb_substrings):
+            return IlInvestmentPolicyMeetingType.IIPB
+        return NOT_CLASSIFIED
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        link_text = item.xpath("a/text()[normalize-space(.)]").get()
+        return datetime.combine(self._parse_date(link_text), time(13, 30))
+
+    @staticmethod
+    def _parse_time_notes(item):
+        """Parse any additional notes on the timing of the meeting"""
+        return "The meeting start time may be earlier than 1:30pm when multiple meetings are " \
```

Comment:
From looking at some of the minutes it seems like meetings are almost always on the same day. It might make sense to have a more generic note here like "Confirm start time with agency" and then maybe default the committees to an earlier time. Not sure if you're seeing a different pattern though, so I'm open to suggestions! Ideally we'd want to list a time earlier rather than later